lib/api/utilities/functions.php

```diff
@@ -6,22 +6,26 @@
  * loaded automatically when the Beans framework is included.
  *
  * @package API\Utilities
+ *
+ * @since   1.5.0
  */
 
+// Load the dependencies.
+require_once dirname( __FILE__ ) . '/polyfills.php';
+
 /**
  * Calls function given by the first parameter and passes the remaining parameters as arguments.
  *
  * The main purpose of this function is to store the content echoed by a function in a variable.
  *
  * @since 1.0.0
  *
- * @param callback $callback The callback to be called.
- * @param mixed    $var      Additional parameters to be passed to the callback.
+ * @param Callable $callback The callback to be called.
+ * @param mixed    $args,... (Optional) Additional parameters to be passed to the callback.
```

Comment:
`(Optional)` format doesn't comply to WP code standards (see [here](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods)) which may be important for the automatic parser. It should be `Optional.`

NB: this comment applies throughout this PR.

Reply:
AFAIK, WPCS (sniffs) doesn't check for this `Optional.` format specifically, but it could do in the future, since it is in the Handbook, and if it's being added, you may as well add it in the format that Core does.

Reply:
@GaryJones the entire [Code Reference](http://www.getbeans.io/code-reference/) is automatically created/maintained using WP Parser, I am not referring to WPCS ;-)